stack.yaml

```diff
@@ -6,6 +6,7 @@ packages:
 
 extra-deps:
   - optparse-applicative-0.15.0.0
+  - relude-0.6.0.0
```

Comment:
I've updated the boundaries in the `.cabal` file. I decided to bump up the lower bound because I'm using `mapMaybeM` function from a new version (it's very helpful :tada:) and `summon` simply won't compile with `relude-0.5.0` :disappointed: In general I don't mind supporting both versions but here I had to increase lower bound...